plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/project/ProjectUtils.java

```diff
@@ -127,6 +127,7 @@
     progress.setWorkRemaining(10 * projectFiles.size() + 10);
     List<IProject> projects = new ArrayList<>(projectFiles.size());
     for (IPath projectFile : projectFiles) {
+      System.out.println("Importing project from: " + projectFile);
```

Comment:
I thought I put that originally in the PR, but this isn't intended to be committed.  Created the PR in the hope that kokoro would pick up the change.